index.js

```diff
@@ -20,8 +21,29 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
-  })
+    Recipe.create({title:'Cold Chocolate Drink', level:'Easy Peasy', ingredients: ['oat milk','raw cacao'], cuisine:'Raw Foods', dishType: 'drink', duration: 2, creator: 'LWB'})
+      .then(recipe => console.log(`The recipe was saved and its title is: ${recipe.title}`))
+      .catch(error => console.log(`An error happened: ${error}`));
+    
+    Recipe.insertMany(data)
+      .then (recipes => recipes.forEach(recipe => console.log(`Title: ${recipe.title}`)))
+      .catch (error => console.log(`Error occurred adding recipes to database:${error}`));
+      
+    Recipe.findOneAndUpdate({title:'Rigatoni alla Genovese'},{$set:{duration:100}}, {new:true})
```

Comment:
Your findOneAndUpdate already knows that the second key is the one to update once it recognises it, so your $set is unnecessary here ($set usually adds new fields to a document; it can also update, but it is used more with the method .aggregate())